### PR TITLE
feat(navigation): Added background overlay for navigation and busines…

### LINKED ISSUE #104: Add background overlay for mobile view

### PR DESCRIPTION
**Pull request by frontendherodk**
…s partner menu

Fixes #104

### CODE REVIEW

**Comment by dami-gg** [comment]
Now I wonder whether the overlay has to be totally dark or it's possible to see some of the background as you implemented it.

File: src/app/navigation.scss [hunk 1]
```diff
@@ -98,43 +86,48 @@
...
     }
-
     &--menu {
       line-height: 2rem;
     }
   }
 }
 
+.placeholder {
+  background-color: rgba(74, 74, 74, 0.7);
```
Comment (dami-gg) [inline src/app/navigation.scss]: Can we use a dress code color here?

File: src/app/navigation.scss [hunk 2]
```diff
@@ -98,43 +86,48 @@
...
+  width: 100%;
+  height: 100%;
+  position: absolute;
+  @media(min-width: $dc-huge-width) {
+    display: none;
+  }
+}
 
+.placeholder.hidden {
```
Comment (dami-gg) [inline src/app/navigation.scss]: You can take advantage of Sass and nest this into the former element block for finding it better

File: src/app/navigation.scss (hunk 1 above)
Comment (frontendherodk) [inline src/app/navigation.scss]: We don't really have rbga colors in dress code. Technically we could, but Im not sure it's worth the implementation

Comment (frontendherodk) [inline src/app/navigation.scss]: We could think about adding a function to Dress Code that converts a hex to a RGBA and vice versa

File: src/app/navigation.scss (hunk 2 above)
Comment (frontendherodk) [inline src/app/navigation.scss]: Valid point. Updated!

**Comment by frontendherodk** [comment]
Correct. It's not part of this issue and should be discussed with UX. I'll open another ticket for this.

**Comment by dami-gg** [comment]
:+1:

**Comment by frontendherodk** [comment]
👍

**Pull request merged**